.changelog/v0.12.0/summary.md

```diff
@@ -0,0 +1,13 @@
+This release notably brings compatibility with Cosmos SDK 0.45 and IBC v3.0.0-rc.0,
```

Comment:
```suggestion
This release notably brings compatibility with Cosmos SDK 0.45 and IBC v3.0.0,
```

Since the proto/interfaces will not change between rc-0 and .0, we can skip that detail.